bpf/complexity-tests/510/bpf_host/6.txt

```diff
@@ -37,3 +37,7 @@
 -DENABLE_IPSEC=1
 -DEVENTS_MAP_RATE_LIMIT_PER_SECOND=1000
 -DENABLE_LOCAL_REDIRECT_POLICY=1
+-DENABLE_SERVICE_PROTOCOL_DIFFERENTIATION=1
```

Comment:
I think this one would be good to have regardless of whether it reproduces the reported issue.

Reply:
Ack! I'll move the PR out of draft.